tests/zkevm/test_worst_compute.py

```diff
@@ -105,7 +114,93 @@ def test_worst_keccak(
     )
 
 
-@pytest.mark.zkevm
+@pytest.mark.valid_from("Cancun")
+@pytest.mark.parametrize(
+    "gas_limit",
+    [
+        Environment().gas_limit,
+    ],
+)
+@pytest.mark.parametrize(
+    "address,static_cost,per_word_dynamic_cost,bytes_per_unit_of_work",
+    [
+        pytest.param(0x02, 60, 12, 64, id="SHA2-256"),
+        pytest.param(0x03, 600, 120, 64, id="RIPEMD-160"),
+        pytest.param(0x04, 15, 3, 1, id="IDENTITY"),
```

Comment:
`bytes_per_unit_of_work` is mainly a concept of for how many bytes of input the precompile has to do "computational work". In the case of hashes, is related to the block size of the compression function. For identity is "1" since work is linear (i.e., not "chunked")

cc @kevaundray